src/Foundation/Http/Controllers/Tools/FileFunctionsService.php

```diff
@@ -166,13 +173,14 @@ static function compress_png($path_to_png_file, $max_quality = 90)
     }
 
     /**
-     * Optimizes JPG file with jpg-recompress
+     * Optimizes JPG file with jpg-recompress.
+     *
+     * @param [type] $path_to_jpg_file [description]
+     * @param int    $max_quality      [description]
      *
-     * @param  [type]  $path_to_jpg_file [description]
-     * @param  integer $max_quality [description]
-     * @return [type]                    [description]
+     * @return [type] [description]
      */
-    static function compress_jpg($path_to_jpg_file, $max_quality = 90)
+    public static function compress_jpg($path_to_jpg_file, $max_quality = 90)
```

Comment:
__[code_sniffer]__
* Method name "FileFunctionsService::compress_jpg" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->